### PR TITLE
Feature/home layout loading

### PR DESCRIPTION
**Pull request by SimonSimCity**
Solves #64 to quite an extend - not fully yet ... someone with experience in tailwind should probably have a closer look at my work and do some follow-up changes here - I just can't get comfortable with it ... 😅 

What's still open to fully solve #64:

- [ ] Find out how to get the age of a person
- [x] Reload when locale changes
- [x] Fix styling for `album` and `playlist`
- [x] Confirm design for `track`
- [x] Implement design, route and component for `contributor` and `podcasts` (should go into a separate issue; both are very similar to a curated playlist) - #132, #133

### CODE REVIEW

**Review by sifferhans — APPROVED** [review]
Looks good to me. 
Nice improvement to i18n too :)

**Comment by SimonSimCity** [comment]
> Nice improvement to i18n too :) Should probably be a separate PR if we were to be that strict, but no matter

IMO not, because it was needed for this PR: https://github.com/bcc-code/bmm-web/pull/129/files#diff-02281a80fab758cb4e8e8359b222a8a5afeaa9d5a7ba858f15d852f1f8969ecfR9

File: pages/index.vue [hunk 1]
```diff
@@ -1,57 +1,104 @@
...
+            <PageHeading v-if="item.type === 'section_header'" :level="3"
+              ><a v-if="typeof item.link === 'string'" :href="item.link"
+                >{{ item.title }}
+              </a>
+              <span v-else>{{ item.title }}</span></PageHeading
+            >
+            <NuxtLink
+              v-else-if="item.type === 'album'"
+              :to="{ name: 'album-id', params: { id: item.id } }"
+              style="display: inline-block; padding: 0 20px 20px 0"
```
Comment (sifferhans) [inline pages/index.vue:67]: Here you should use `class="inline-block pr-4 pb-4"` instead of inline styles

File: pages/index.vue [hunk 2]
```diff
@@ -1,57 +1,104 @@
...
+            >
+              <ProtectedImage
+                :src="item.cover || ''"
+                style="
+                  max-width: 214px;
+                  border: 1px solid rgba(129, 136, 143, 0.1);
+                  border-radius: 16px;
```
Comment (sifferhans) [inline pages/index.vue:74]: Probably something like `class="max-w-[214px] border rounded-md"`

File: pages/index.vue (hunk 1 above)
Comment (SimonSimCity) [reply pages/index.vue:67]: This really looks like writing styles in the css property 🤪😂

File: pages/index.vue (hunk 2 above)
Comment (SimonSimCity) [reply pages/index.vue:74]: Why use `rounded-md` here? Isn't the `-md` equal to `0.375rem`, which is `6px` for most systems..? And adding `border` here gives it a solid black border, not the color I got in the design.

**Comment by SimonSimCity** [comment]
I've changed quite some stuff for reusability, added grouping and also added a well visible note `Not yet implemented` for Podcasts (even though its visibly correct) and non-implmeneted types as contributors.

**Review by sifferhans — APPROVED** [review]

**Pull request merged**